ElementX/Sources/Other/ScrollViewAdapter.swift

```diff
@@ -49,11 +49,19 @@ class ScrollViewAdapter: NSObject, UIScrollViewDelegate {
             .eraseToAnyPublisher()
     }
     
+    private let scrollViewIsAtTopEdge: CurrentValueSubject<Bool, Never> = .init(false)
+    var scrollViewIsAtTopEdgePublisher: CurrentValuePublisher<Bool, Never> {
```

Comment:
Looking at the ones above, this would be more aligned naming
```suggestion
    private let isAtTopEdgeSubject: CurrentValueSubject<Bool, Never> = .init(false)
    var isAtTopEdge: CurrentValuePublisher<Bool, Never> {
```